### ISSUE TITLE
Tooltip keyboard accessibility (AVT 2)

### ISSUE THREAD

**Issue by johnharan** [IBM/carbon-components-angular#263]
The tooltip does not show on keyboard focus, only on mouse hover.

> Guideline 2.1 Keyboard Accessible: Make all functionality available from a keyboard.
https://www.ibm.com/able/guidelines/ci162/accessibility_checklist.html

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> What browser are you working in?

chrome Version 70.0.3538.77

> What version of the Carbon Design System are you using?

"carbon-components": "9.48.0",
"carbon-components-angular": "2.0.6",

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IWR
